module/surveys/helper.js

```diff
@@ -1413,6 +1415,208 @@ module.exports = class SurveysHelper {
     });
   }
 
+  /**
+   * List of surveys based on UserId.
+   * @method
+   * @name userSurvey
+   * @param {String} requestUserId - id of the user to fetch docmunet.
+   * @param {String} queryType -     type of collection to query whether its survey or obs
+   * @returns {JSON} List or count of surveys for specific user.
+   */
+
+  static userSurvey(requestUserId, queryType = "") {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let surveyListedBasedOnStatus = [];
+        let bigNumberOfSurvey;
+        //if there is an queryType
+        if (queryType != "") {
```

Comment:
instead of queryType use stats it should be boolean and if its true then it should return bignumber @praveenKDass